package.json

```diff
@@ -4,14 +4,26 @@
   "private": true,
   "description": "Frontend build tools and development utilities for Kolibri Studio",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
     "lint:py": "flake8 --exit-zero",
     "makemessages": "node ./extract_frontend_messages.js ./contentcuration/contentcuration/static/js/edit_channel/**{/*,}.{js,vue}",
     "build": "webpack --config webpack.config.js",
+    "cypress:run": "wait-on -t 20000 tcp:8081 && cypress run --spec cypress/integration/login.js",
+    "cypress:test": "yarn run build && npm-run-all --parallel --race testserver cypress:run",
+    "pytest": "cd contentcuration; pytest",
+    "pytest:reusedb": "cd contentcuration; pytest --reuse-db",
+    "postgres": "service postgresql start || pg_ctl -D /usr/local/var/postgresql@9.6 start || true",
+    "redis": "service redis-server start || redis-server /usr/local/etc/redis.conf || true",
+    "services": "npm-run-all -c --parallel --silent minio redis postgres",
+    "unittests": "npm-run-all --parallel --race services pytest",
```

Comment:
What does `--race` here mean?

Reply:
It's a bit of a misnomer IMHO, but what it means is that npm-run-all will exit once one of the commands returns a 0 status code. So basically, it ensures that the command completes and returns an exit code even if the services command never returns.